UNIT_TESTS.md

```diff
@@ -0,0 +1,82 @@
+# Unit Tests
+
+In addition to the code of the module itself, it is important to include a suite of
+tests. They guarantee that your add-on will remain functional when changes are
+applied to the code on which it depends. Once committed to GitHub, the unit
+tests are automatically run by the CI/CD pipeline every time the code in the
+repository is modified. Notifications are issued if a modification somehow
+affects your add-on.
+
+## Requirements
+
+The unit tests depend on the GRASS development package. It might already be
+installed on your system, but make sure. On a packaged operating system like
+Ubuntu or Debian Linux this can be achieved with:
+
+```bash
+sudo apt install grass-dev
+```
```

Comment:
```suggestion
installed on your system, but make it sure. On a packaged operating system like
Ubuntu or Debian Linux this can be achieved with:

```bash
sudo apt install grass-dev
```

and on Fedora/Centos with:

```bash
sudo dnf install grass-devel
```

```

Reply:
Address in 8f5b672576d1b085c64d207d22b9a75daa331c54